src/onnx/onnx.cpp

```diff
@@ -325,16 +325,32 @@ struct onnx_parser
                                const attribute_map& attributes,
                                std::vector<instruction_ref> args)
     {
-        op::clip op;
+        auto input_lens = args[0]->get_shape().lens();
+        instruction_ref min_arg;
+        instruction_ref max_arg;
+        if(args.size() > 1)
+        {
+            min_arg = args[1];
+
+            if(args.size() > 2)
+            {
+                max_arg = args[2];
+            }
+        }
         if(contains(attributes, "max"))
         {
-            op.max_val = parse_value(attributes.at("max")).at<float>();
+            float max_val = parse_value(attributes.at("max")).at<float>();
+            max_arg       = prog.add_literal(max_val);
         }
         if(contains(attributes, "min"))
         {
-            op.min_val = parse_value(attributes.at("min")).at<float>();
+            float min_val = parse_value(attributes.at("min")).at<float>();
+            min_arg       = prog.add_literal(min_val);
         }
-        return prog.add_instruction(op, std::move(args));
+        min_arg = prog.add_instruction(op::multibroadcast{input_lens}, min_arg);
```

Comment:
I am wondering why we need to multibroadcast? We can just use the index 0 to get the min/max value, right?

Reply:
I added a multibroadcast to reuse nary() in the device function without adding more complexity.